src/components/atoms/Table/Components/TableHeader.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { FunctionComponent, ReactNode } from "react";
+
+export type TableHeaderProps = {
+  className?: string,
+  children: ReactNode,
+}
+
+const TableHeader: FunctionComponent<TableHeaderProps> = (props) => {
```

Comment:
`TableHeader = (props: TableHeaderProps)`
FunctionComponent is discouraged. [Why ?](https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components)